usage/jsgui/src/main/webapp/assets/js/router.js

```diff
@@ -18,14 +18,15 @@
 */
 define([
     "brooklyn", "underscore", "jquery", "backbone",
+    "codemirror",
```

Comment:
this is what is breaking the run -- we don't use it locally so we can remove it I think; but if it's added of course there needs to be a corresponding `CodeMirror` argument to the following `function`